examples/1_hw_basic_control.py

```diff
@@ -17,8 +17,8 @@
     print(f"camera_pose={camera_pose}")
 
     # Move camera
-    robot.head.set_pan_tilt(pan=np.pi / 4, tilt=-np.pi / 3, blocking=True)
-    robot.head.look_at_ee(blocking=True)
+    robot.head.set_pan_tilt(pan=np.pi / 4, tilt=-np.pi / 3)
```

Comment:
i generally prefer using constants here, although this is an example so not sure

Reply:
Yeah seems like overkill for examples. I could define constants like DESIRED_TILT and stuff but I don't think it makes the code any more readable.